### PR TITLE
crex24 - fetchTradingFees

### PR DESCRIPTION
**Pull request by pcriadoperez**
Note: I tested only public method. Please test private method before merging
` node examples/js/cli.js crex24 fetchTradingFees "{'method':'fetchPrivateTradingFees'}"

### CODE REVIEW

**Comment by frosty00** [comment]
i think the info should go on the outside of the result, similar to how we do everywhere with `fetchBalance` instead of repeating the same `info` multiple times

**Comment by kroitor** [comment]
> i think the info should go on the outside of the result, similar to how we do everywhere with `fetchBalance` instead of repeating the same `info` multiple times

I think that would be wrong, cause this response structure is closer to fetchTickers (indexed by symbol), rather than fetchBalance. The tickers-response (market info) is indexed by symbols. The semantic structure of the fees-response is the same, also market info indexed by symbols.

**Comment by frosty00** [comment]
in this case we're just repeating the same info from the response, so it doesn't really make sense to duplicate that data

**Comment by frosty00** [comment]
i would agree if the info was some piece of data relevant to each market

**Comment by kroitor** [comment]
> i would agree if the info was some piece of data relevant to each market

Sometimes, the exchange response has multiple fees per market, other times it has just one fee for all markets. The purpose of `info` is to contain the original source of data used to derive the structure that contains the `info`.

**Comment by kroitor** [comment]
@frosty00 we could introduce a method called `fetchExchangeTradingFee`, that would return just one common maker/taker fee and just one `info`. However, if you think from the users' perspective, that would be a lot less convenient to use than repeating the `info` in `fetchTradingFees()`, cause the user will have to differentiate between the exchanges' available methods and their formats. Instead, we offer the same format returned from `fetchTradingFees()` from all exchanges.

File: js/crex24.js [hunk 1]
```diff
@@ -754,6 +757,113 @@ module.exports = class crex24 extends Exchange {
...
+                'percentage': true,
+                'tierBased': true,
+            };
+        }
+        return result;
+    }
+
+    async fetchPrivateTradingFees (params = {}) {
+        await this.loadMarkets ();
+        const response = await this.TradingGetTradingFee (params);
```
Comment (kroitor) [inline js/crex24.js:829]: A typo here: `TradingGetTradingFee` > `tradingGetTradingFee`.

**Pull request merged**